aws-cpp-sdk-transfer/include/aws/transfer/TransferManager.h

```diff
@@ -60,6 +60,10 @@ namespace Aws
              * It is not a bug to use the same executor, but at least be aware that this is how the manager will be used.
              */
             Aws::Utils::Threading::Executor* transferExecutor;
+            /**
+             * If this is set to true, calculate content-md5 and set it to upload part request.
+             */
+            bool needsComputeContentMd5;
```

Comment:
Please, change the naming of the variable and the documentation to the following.

```cpp
/**
 * When true, TransferManager will calculate the MD5 digest of the content being uploaded.
 * The digest is sent to S3 via an HTTP header enabling the service to perform integrity checks.
 * This option is disabled by default.
 */
bool computeContentMD5;
```